Tests/ML/util.py

```diff
@@ -277,7 +277,7 @@ def get_default_workspace() -> Workspace:
 
 
 def model_train_unittest(config: Optional[DeepLearningConfig],
-                         dirs: OutputFolderForTests,
+                         dirs: Union[OutputFolderForTests, Path],
```

Comment:
dirs represents a single directory I think? should it be called 'dir' (singular) instead?

Reply:
Well maybe not dir because thats a python function but something else singular!